libafl_v8/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "libafl_v8"
+version = "0.8.1"
+authors = ["Addison Crump <research@addisoncrump.info>"]
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+ahash = { version = "0.7", features = ["compile-time-rng"] }
+deno_ast = { version = "0.17.0", features = ["transpiling"] }
+# fetch deno directly because of git deps; cannot use local submodule due to workspacing
```

Comment:
Does this comment still apply? I've tried switching to deno from crates.io ([here](https://github.com/Mrmaxmeier/LibAFL/commit/c12ad85cab5d7b7a12e2e63242078d1b438d2d60)) and that seems to work fine.